store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -50,6 +30,13 @@
 import org.apache.rocketmq.store.ha.HAService;
 import org.apache.rocketmq.store.schedule.ScheduleMessageService;
 
+import java.net.*;
```

Comment:
It is not recommended to import all classes with wildcard.

Reply:
+1